Backend/src/main/java/idealab/api/operations/DropboxOperations.java

```diff
@@ -99,10 +110,63 @@ public void deleteDropboxFile(PrintJob printJob){
     // 1. Delete existing dropbox file using deleteDropboxFile() method
     // Note:  if the file path does not start with a "/" then there is either an error or it was deleted already.
     if(printJob.getDropboxPath().startsWith("/")){
-      deleteDropboxFile(printJob);
+      deleteDropboxFile(printJob.getDropboxPath());
     }
     // 2. Create new sharable data link using uploadDropboxFile() method
     LocalDateTime currentTime = LocalDateTime.now();
     return uploadDropboxFile(currentTime.toLocalTime().toNanoOfDay(), file);
   }
+
+  public PrintJobResponse updateDropboxPath(DropBoxFilePathRequest request) {
+    request.validate();
+
+    PrintJob printJob = printJobRepo.findPrintJobById(request.getPrintJobId());
+
+    if(printJob == null)
+      throw new IdeaLabApiException(PRINT_JOB_CANT_FIND_BY_ID);
+
+    LocalDateTime currentTime = LocalDateTime.now();
+    String newPath = "/" + currentTime.toLocalTime().toNanoOfDay() + request.getNewPath();
```

Comment:
After time may want to add a "-" before the getNewPath().  The files now look like 00000newPath instead of 0000-newPath.

Also may want to make sure the new path has '.stl' at the end, I noticed I could change the file name to either 'newPath' or 'newPath.stl.'

Reply:
made issue #203